FluentUI.Demo/src/androidTest/java/com/microsoft/fluentuidemo/demos/V2BottomDrawerUITest.kt

```diff
@@ -241,4 +242,18 @@ class V2BottomDrawerUITest : BaseTest() {
         }
         closeCheckForVerticalDrawer()
     }
+    @Test
+    fun testNonExpandableAndExpandDrawerClicked() {
+        // Set expandable = false
+        composeTestRule.onNodeWithText(getString(R.string.drawer_expandable), useUnmergedTree = true).performClick()
+        // Click on the "Open Drawer" button
+        composeTestRule.onNodeWithText(getString(R.string.drawer_expand)).performClick()
```

Comment:
this is called below as well. remove it.

Reply:
Should remove this line instead of below.

Reply:
Done